ocaml/xapi/helpers.ml

```diff
@@ -1312,3 +1312,7 @@ let retry ~__context ~doc ?(policy = Policy.standard) f =
 
 let retry_with_global_lock ~__context ~doc ?policy f =
   retry ~__context ~doc ?policy (fun () -> with_global_lock f)
+
+let vdi_operation_can_be_performed_live op = match op with
+  | `snapshot | `resize_online | `blocked | `clone | `mirror | `copy -> true
```

Comment:
Hmm I'm not sure about that, I have pinged @jonludlam for help in ring3 channel.